mandatory/1-fix-functions.js

```diff
@@ -10,18 +10,18 @@
 
 */
 
-function mood() {
-  let isHappy = true;
```

Comment:
I see you have taken out the boolean statement that this function needs. If you run this function, you will get an error. isHappy= true; has to be included for this function to run-I suggest that you readd the line: 
let isHappy = true;

Reply:
when you have a statement without assigning it a value, the value "undefined is assigned".